client_code/PageBreak/form_template.yaml

```diff
@@ -1,12 +1,9 @@
 container:
   type: HtmlTemplate
   properties: {tooltip: '', background: '', foreground: '', border: '', visible: true,
-    role: null, html: "<!--\nBased on the snippet at https://anvil.works/forum/t/plots-in-pdf-being-divided-between-two-pages/7774/5\n\
-      -->\n<div class=\"break-container\" style=\"overflow: hidden;\">\n  <div style=\"\
-      page-break-after:always;\"/>\n  <div style=\"margin-top: -55px;\"/>\n</div>\n\
-      \n<style>\n  .break-container {\n    border: 1px solid grey;\n  }\n  @media\
-      \ print {\n    .break-container {\n      border: none;\n    }\n  }\n</style>"}
+    role: null, html: ''}
```

Comment:
If you go with the above version I'd leave the html and css in - it makes it nicer in the design view at least.